test/unit/mdc-select/mdc-select.test.js

```diff
@@ -214,17 +278,17 @@ test('adapter#focus focuses on the root element', () => {
 });
 
 test('adapter#makeTabbable sets the root element\'s tabindex to 0', () => {
-  const {component, fixture} = setupTest();
-  fixture.tabIndex = -1;
+  const {component, menuEl} = setupTest();
+  menuEl.tabIndex = -1;
   component.getDefaultFoundation().adapter_.makeTabbable();
-  assert.equal(fixture.tabIndex, 0);
+  assert.equal(menuEl.tabIndex, 0);
 });
 
 test('adapter#makeUntabbable sets the root element\'s tabindex to -1', () => {
-  const {component, fixture} = setupTest();
-  fixture.tabIndex = 0;
+  const {component, menuEl} = setupTest();
+  menuEl.tabIndex = 0;
   component.getDefaultFoundation().adapter_.makeUntabbable();
-  assert.equal(fixture.tabIndex, -1);
+  assert.equal(menuEl.tabIndex, -1);
 });
 
 test('adapter#getComputedStyleValue gets the computed style value of the prop from the root element', () => {
```

Comment:
I'm talking about the `getComputedStyleValue` test, which doesn't show any changes in the diff and still references `fixture`?  I know you changed the `setStyle` test, but you changed both of these adapter methods.